src/memory.rs

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 IBM Corp.
+
+use crate::collector::*;
+use regex::Regex;
+
+// add -h to get human readable sizes, else in bytes
+const MEM_COMMAND: &'static str = "free";
+
+// would be nice if we only had to run `$ free` once
+// must run several times since we can only add one collector entry per function call
```

Comment:
Not accurate, we cache exact command results, so `free` is actually called only once